schemas/context/viewability-details.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/viewability-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Viewability Details",
```

Comment:
Maybe we should change the title (just the title, not the schema name) to "MRC Viewability"